HHS/Head-Start-TTADP
# Remove cookie timeout and enable CI axe scan

**Description of change**
 * Axe scans the page right after load. In the admin page there is a fake delay to simulate loading data from the API. This delay causes axe to run against the half loaded page. Added a delay between fetching of the page and the axe run to allow the page to fully load. See screenshots below.

With no delay:
![screenshot](https://user-images.githubusercontent.com/6631790/101936773-97a95900-3ba6-11eb-80b4-5fd135fc2106.png)

With a second delay:
![screenshot2](https://user-images.githubusercontent.com/6631790/101936844-b1e33700-3ba6-11eb-96fc-c733a52d00e8.png)

 * Having no cookie timeout defined causes the cookie to expire only when the user closes their browser, instead of their cookie expiring after a set amount of time. This prevents an edge case where the user is active on the site without hitting the server and having their session expire on the backend but not the frontend.

**How to test**
#159
1) Pull down, and login.
2) Look at the cookie sent from the server, it should be a listed as a `session` cookie in the `expires at` field.

#132
1) Verify the build passed and the `accessibility_scan` [section has no errors](https://app.circleci.com/pipelines/github/adhocteam/Head-Start-TTADP/734/workflows/f0e4b075-afe9-4741-9a44-16bf377f392a/jobs/3344)

**Issue(s)**
* closes https://github.com/HHS/Head-Start-TTADP/issues/159
* closes https://github.com/HHS/Head-Start-TTADP/issues/132

**Checklist**

- [n/a] Code tested
- [n/a] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
